### PR TITLE
Typeid serialization change

### PR DESCRIPTION
**Pull request by xen-0**

### CODE REVIEW

**Comment by GDYendell** [comment]
Do you need the Mutators to have typeid instead of type, too?

File: docs/serialisation.rst [hunk 1]
```diff
@@ -8,7 +8,7 @@ for the CompoundGenerator is as follows::
         type: "CompoundGenerator",
         generators: [
             {
-                type: "LineGenerator"
+                typeid: "LineGenerator"
```
Comment (xen-0) [inline docs/serialisation.rst]: Not scanpointgenerator:generator/LineGenerator:1.0?

File: scanpointgenerator/generator.py [hunk 2]
```diff
@@ -66,6 +66,7 @@ def register_subclass(cls, generator_type):
         def decorator(generator):
 
             cls._generator_lookup[generator_type] = generator
```
Comment (GDYendell) [inline scanpointgenerator/generator.py]: Generator.from_dict() needs to do d['typeid'] instead of d['type'].

**Comment by GDYendell** [comment]
Looks good :+1:

**Pull request merged**